pure2.md

```diff
@@ -0,0 +1,535 @@
+
+    -   The latest version of the Snapshot metadata file at the time the offline update was generated
+
+-   Director repository metadata:
+
+    -   All Root metadata files from the Director repository, from version 1 to the most-recent version at the time the offline update is generated
+
+    -   The offline-update Snapshot metadata file
+
+    -   Exactly one offline-update targets metadata file
+
+-   All target images referenced in the offline-update targets metadata file
+
+### Metadata verification procedures
+
+The procedures for verifying offline-update metadata are similar to the procedures for verifying normal director metadata. However, there are some key differences. The procedures for sending images from the primary to secondaries, as well as for verifying and installing the images once the metadata has been verified, remain the same.
+
+Note: wherever the "Uptane Standard" or "the Standard" is referenced, it refers to version 1.2.0 of the Standard, available at https://uptane.github.io/uptane-standard/1.2.0/uptane-standard.html.
+
+The primary still SHALL verify all target images listed in the Director offline-update metadata, while secondaries MAY verify only the images they will install.
+
+#### Download and verify metadata from the well-known location
+
+The Primary SHALL download metadata for all offline-update targets and perform a full verification on it. Full verification of metadata means that the ECU checks that the Targets metadata about images from the Director repository matches the Targets metadata about the same images from the Image repository. This provides resilience to a key compromise in the system.
+
+Because of the nature of offline updates, sometimes the offline-update image will contain newer metadata than the primary has, and sometimes the converse will be true. Most of the differences or new steps in the verification process are to deal with this reality; the rest of the differences are steps that were removed because they aren't applicable, mostly relating to the ECU version reports and vehicle version manifest.
+
+In order to perform offline-update verification, an ECU SHALL perform the following steps.
+
+1.  Load and verify the current time or the most recent securely attested time.
+
+2.  Check if updated Director root metadata is available in the well-known location, following the procedure listed in Section 5.4.4.3 of the Standard, loading the root metadata file(s) from the well-known location instead of attempting to download them from the repository.
+
+3.  Update offline-update Snapshot metadata from the well-known location:
+
+    1.  Load the offline-update Snapshot metadata file from the well-known location. If the version number of the existing offline-update Snapshot metadata file is equal to or greater than the one found in the well-known location, continue to step 4.
```

Comment:
"existing"? You mean previously stored? The phrasing here is ambiguous because we're talking about two local storage locations, so we should be really clear about what to get from where.

Reply:
Fixed